src/components/Resource/Forms/Training/TrainingSubmit.tsx

```diff
@@ -391,10 +401,24 @@ export const TrainingFormSubmit = ({ model }: { model: NonNullable<TrainingModel
     //   });
     // }
 
-    form.setValue('targetSteps', newSteps);
+    if (form.getValues('targetSteps') !== newSteps) {
+      form.setValue('targetSteps', newSteps);
+    }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [watchFields]);
 
+  useEffect(() => {
+    const eta = calcEta(
+      form.getValues('networkDim'),
+      form.getValues('networkAlpha'),
+      form.getValues('targetSteps'),
+      formBaseModel
+    );
+    const price = eta !== undefined ? calcBuzzFromEta(eta) : eta;
+    setBuzzCost(price);
```

Comment:
This seems to be a bit wrong, since we will always charge for training, right? 😛 Maybe fallback to a base cost?

Reply:
this will implicitly never happen, as you can only change the advanced settings when the model has been selected. even if they do, there is a backend check. eta being undefined means something is wrong, so we don't want to fallback to a base price.